src/spec-node/featuresCLI/package.ts

```diff
@@ -0,0 +1,119 @@
+import path from 'path';
+import { Argv } from 'yargs';
+import { CLIHost, getCLIHost } from '../../spec-common/cliHost';
+import { loadNativeModule } from '../../spec-common/commonUtils';
+import { Log, LogLevel, mapLogLevel } from '../../spec-utils/log';
+import { isLocalFile, isLocalFolder, mkdirpLocal, rmLocal } from '../../spec-utils/pfs';
+import { createLog } from '../devContainers';
+import { UnpackArgv } from '../devContainersSpecCLI';
+import { getPackageConfig } from '../utils';
+import { doFeaturesPackageCommand } from './packageCommandImpl';
+
+const targetPositionalDescription = `
+Package features at provided [target] (default is cwd), where [target] is either:
+   1. A collection folder containing a './src' folder with [1..n] features.
```

Comment:
Why not supply the src-folder's path directly? This would avoid the requirement that the features are located in a `src` folder (e.g., they could be in a subfolder of `src` or the folder could be named differently like `features-src` or the repo root could contain the feature folders directly).

Reply:
I was attempting to keep consistent with `features test ...`, which expects that folder layout.  I thought it may be less confusion to always operate from the "root" of the collection (this would be the root of the `devcontainers/features` repo, for example.

Reply:
i've made the change to directly expect the src folder, whereever that may be.

Reply:
Maybe align `features test` with this then?

Reply:
The way I have it wired up, `features test` looks for two mirrored folders `src` and `test`.  An improvement there is certainly to let a user indicate both the src and test folders independently, in which case it would line up better with this command.  

Either way, not a huge issue and simple enough 👍